tests/lib/lore_test.sh

```diff
@@ -14,6 +14,31 @@ function oneTimeSetUp()
   cp "${SAMPLES_DIR}/web/reduced_lore_main_page.html" "${CACHE_LORE_DIR}/lore_main_page.html"
 }
 
+function setUp()
+{
+  export ORIGINAL_PATH="$PWD"
+
+  export BOOKMARKED_SERIES_PATH="${SHUNIT_TMPDIR}/lore_bookmarked_series"
+
+  touch "${BOOKMARKED_SERIES_PATH}"
+
+  cd "${SHUNIT_TMPDIR}" || {
+    fail "($LINENO): setUp(): It was not possible to move into ${SHUNIT_TMPDIR}"
+    return
+  }
+}
+
+function tearDown()
+{
+  # Empty file
+  : > "${BOOKMARKED_SERIES_PATH}"
```

Comment:
You can delete this file in the `tearDown` instead of empty it; I believe this way is more readable.

Reply:
Right! I chose this approach because an error was thrown when trying to delete it (cause the file didn't exist). Thinking now, it is most likely because of the command on the comment right below 🤣